yt_dlp/extractor/nebula.py

```diff
@@ -478,3 +481,48 @@ def _real_extract(self, url):
             playlist_id=collection_slug,
             playlist_title=channel.get('title'),
             playlist_description=channel.get('description'))
+
+
+class NebulaSeasonIE(NebulaBaseIE):
+    IE_NAME = 'nebula:season'
+    _VALID_URL = rf'{_BASE_URL_RE}/(?P<series>[\w-]+)/?season/(?P<season_number>\d+)'
+    _TESTS = [{
+        'url': 'https://nebula.tv/jetlag/season/15',
+        'info_dict': {
+            'id': 'jetlag_15',
+        },
+        'playlist_count': 8,
+    }, {
+        'url': 'https://nebula.tv/jetlag/season/14',
+        'info_dict': {
+            'id': 'jetlag_14',
+        },
+        'playlist_mincount': 8,
+    }]
+
+    def _entries(self, data):
+        base_url = 'https://nebula.tv'
+        for episode in traverse_obj(data, ('episodes')):
+            episode_id = traverse_obj(episode, ('video', 'id'))
+            metadata = self._extract_video_metadata(episode)
+            yield self.url_result(smuggle_url(f'{base_url}/videos/{episode_id}', {'id': episode_id}),
+                                  NebulaIE, episode_id, url_transparent=True, **metadata)
+        for extra in traverse_obj(data, ('extras')):
+            extra_id = traverse_obj(extra, ('items', ..., 'id'))[0]
+            metadata = self._extract_video_metadata(traverse_obj(extra, ('items', 0)))
+            yield self.url_result(smuggle_url(f'{base_url}/videos/{extra_id}', {'id': extra_id}),
+                                  NebulaIE, extra_id, url_transparent=True, **metadata)
+        for trailer in traverse_obj(data, ('trailers')):
+            trailer_id = trailer.get('id')
+            metadata = self._extract_video_metadata(trailer)
+            yield self.url_result(smuggle_url(f'{base_url}/videos/{trailer_id}', {'id': trailer_id}),
+                                  NebulaIE, trailer_id, url_transparent=True, **metadata)
+
+    def _real_extract(self, url):
+        season_name, season_id = self._match_valid_url(url).group('series', 'season_number')
+        playlist_id = f'{season_name}_{season_id}'
+        data = self._call_api(f'https://content.api.nebula.app/content/{season_name}/season/{season_id}', playlist_id)
+        if not traverse_obj(data, ('episodes'), ('extras'), ('trailers')):
+            traverse_obj(data, ('episodes'), ('extras'), ('trailers'))
+            raise ExtractorError('No Episodes, Outtakes, Trailes Found.')
```

Comment:
- line too long
- `series`
- don't raise here
```suggestion
        data = self._call_api(
            f'https://content.api.nebula.app/content/{series}/season/{season_id}', playlist_id)

```